test/Elastica/Base.php

```diff
@@ -22,41 +22,6 @@ protected static function showDeprecated()
         \error_reporting(\error_reporting() | E_USER_DEPRECATED);
     }
 
-    protected function assertFileDeprecated($file, $deprecationMessage)
```

Comment:
Quick reminder for me: We don't need this anymore because ...?

Reply:
This method was not used